src/test/crimson/perf_crimson_msgr.cc

```diff
@@ -161,13 +175,13 @@ static seastar::future<> run(unsigned rounds,
       }
       seastar::future<> ms_dispatch(ceph::net::ConnectionRef c,
                                     MessageRef m) override {
-        ceph_assert(m->get_type() == CEPH_MSG_OSD_OPREPLY);
+        ceph_assert(m->get_type() == CEPH_MSG_OSD_OP);
```

Comment:
why? i assume the Server should always reply `MOSDOpReply`, no?

Reply:
It's because server needs to reply with MOSDOp to generate write workload.